verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -1442,6 +1442,138 @@ TEST_F(VerilogLanguageServerSymbolTableTest, CheckReferenceUnknownSymbol) {
   ASSERT_EQ(response_b["result"].size(), 0);
 }
 
+struct RenameRequestParams {
+  int id;
+  std::string file;
+  std::string newName;
+  int line;
+  int character;
+};
+
+std::string RenameRequest(RenameRequestParams params) {
+  json renamerq = {
+
+      {"jsonrpc", "2.0"},
+      {"id", params.id},
+      {"method", "textDocument/rename"},
+      {"params",
+       {
+           {"textDocument",
+            {
+                {"uri", params.file},
+            }},
+           {
+               "position",
+               {{"line", params.line}, {"character", params.character}},
+           },
+           {"newName", params.newName},
+       }}};
+  return renamerq.dump();
+}
+std::string PrepareRenameRequest(RenameRequestParams params) {
+  json renamerq = {
+
+      {"jsonrpc", "2.0"},
+      {"id", params.id},
+      {"method", "textDocument/prepareRename"},
+      {"params",
+       {
+           {"textDocument",
+            {
+                {"uri", params.file},
+            }},
+           {
+               "position",
+               {{"line", params.line}, {"character", params.character}},
+           },
+           {"newName", params.newName},
+       }}};
+  return renamerq.dump();
+}
+
+// Runs tests for textDocument/rangeFormatting requests
+TEST_F(VerilogLanguageServerSymbolTableTest, PrepareRenameTest) {
```

Comment:
The naming should reflect what we're testing, e.g. `PrepareRenameReturnsRangeOfEditableSymbol` or something along those lines. Not all tests are named like this yet, but this is helpful when seeing all the test outputs from the `bazel test //...`